src/components/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import { Drawer } from './Drawer';
+import Button from '.././Button/Button';
+
+export default {
+  title: 'Drawer',
+  component: Drawer,
+};
+
+export const persistentDrawer = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div
+      style={{
+        margin: '2.5rem auto 0 auto',
+        width: '500px',
+        lineHeight: '5rem',
+        border: 'solid 1px #D3D3D3',
+        borderRadius: '10px',
+        textAlign: 'center',
```

Comment:
I think it would be better to write the style in a stylesheet than inline, even though it is not component style.